featurebyte/query_graph/model/graph.py

```diff
@@ -115,6 +116,19 @@ def sorted_node_names_by_ref(self) -> List[str]:
             self._update_cache()
         return self._sorted_node_names_by_ref
 
+    @property
+    def sorted_node_names(self) -> List[str]:
```

Comment:
Was this also a critical path that slowed down the graph operations?

Reply:
Yes, when performing the quick prune operation. This is called multiple times.